.github/workflows/generate-lbf-icml.yml

```diff
@@ -1,11 +1,15 @@
-name: Generate ICML for LBF
+name: Generate ICMLs
 
 on:
+  push:
+    branches:
+      - master
   pull_request:
     paths:
-      - 'th/LBF/03_public/*.md'
-      - 'th/LBF/metadata.yaml'
+      - 'th/**/03_public/*.md'
+      - 'th/**/*.yaml'
```

Comment:
@mattleff does this path include the divisions of PP?

Reply:
@bountonw I believe it should, but I can double check on a PR without changes on all the other triggering paths.